website/templates/project/registration_editor_templates.mako

```diff
@@ -154,4 +158,28 @@
     </div>
 </script>
 
+<script>
+ function setAllBoxes(value) {
```

Comment:
Is there a reason not to do this in the KO binding of #authorDialog? You can pass an arbitrary object in as your viewModel.